contracts/src/bridge/L1/ObscuroBridge.sol

```diff
@@ -35,6 +35,10 @@ contract ObscuroBridge is
         _grantRole(NATIVE_TOKEN_ROLE, address(0x0));
     }
 
+    function promoteToAdmin(address newAdmin) external onlyRole(ADMIN_ROLE) {
```

Comment:
So is the intention that the account with PK 4bfe14725e685901c062ccd4e220c61cf9c189897b6c78bd18d7f51291b2b8f1 is promoted to admin at startup and is available for use in the E2E tests?

Reply:
yep